packages/test/src/utils/inferLanguages.test.ts

```diff
@@ -0,0 +1,53 @@
+import { deepEqual } from "assert"
+import {LanguageEntity, serializeLanguages, serializeNodes } from "@lionweb/core"
+import { inferLanguagesFromChunk } from "@lionweb/utilities"
+
+import { libraryExtractionFacade, libraryModel } from "../instances/library.js"
+import { minimalLibraryLanguage } from "../languages/minimal-library.js"
+import { multiExtractionFacade, multiModel } from "../instances/multi.js"
+import { multiLanguage } from "../languages/multi.js"
+
+describe("inferLanguagesFromChunk", () => {
+    it("should correctly infer the minimal library language from the instance", () => {
```

Comment:
Alternatively, we make sure that the example library instance covers the entire library language. That way we wouldn't need to have a minimal library language.

Reply:
I don't think we can infer the entire library language since it involves inheritance which probably cannot be deduced from the instance

Reply:
Ah, true! :)